.github/workflows/score_new_plugins.yml

```diff
@@ -0,0 +1,111 @@
+
+      - name: Get changed files
+        uses: dorny/paths-filter@v2.11.1
+        id: filter
+        with:
+          list-files: shell
+          filters: |
+            changed:
+              - '**'
+      - name: Save changed files to env var
+        run: echo "CHANGED_FILES=${{ steps.filter.outputs.changed_files }}" >> $GITHUB_ENV
+      
+      - name: Parse changed files with python script
+        run: |
+          echo "PLUGIN_INFO=$(python .github/workflows/parse_changed_files.py '${{ env.CHANGED_FILES }}')" >> $GITHUB_ENV
+          
+      - name: Save plugin info to outputs
+        id: getpluginfo
+        run: |
+          echo "PLUGIN_INFO=$PLUGIN_INFO" >> $GITHUB_OUTPUT
+
+      - name: check if scoring needed
+        id: runscore
+        run: |
+          echo "RUN_SCORE=$(jq -r '.run_score' <<< "$PLUGIN_INFO")" >> $GITHUB_OUTPUT
+
+  get_submitter_info:
+    name: Get PR author email and (IF web submission) Brain-Score user ID
+    runs-on: ubuntu-latest
+    needs: changes_models_or_benchmarks
+    if: ${{ needs.changes_models_or_benchmarks.outputs.RUN_SCORE == 'True' }}
+    env:
+      PLUGIN_INFO: ${{ needs.changes_models_or_benchmarks.outputs.PLUGIN_INFO }}
+    outputs:
+      PLUGIN_INFO: ${{ steps.add_email_to_pluginfo.outputs.PLUGIN_INFO }}
+    steps:
+      - name: Parse user ID from PR title (WEB ONLY)
+        id: getuid
+        if: ${{ github.event.pull_request.labels.*.name == 'automerge-web' }}
+        run: |
+          echo "BS_UID="$(<<<${{ github.event.pull_request.title }} | sed -E 's/.*\(user:([^)]+)\).*/\1/'"" >> $GITHUB_ENV
+      - name: Add user ID to PLUGIN_INFO (WEB ONLY)
+        id: add_uid_to_pluginfo
+        if: ${{ github.event.pull_request.labels.*.name == 'automerge-web' }}
+        run: |
+          echo "The Brain-Score user ID is ${{ steps.getuid.outputs.BS_UID }}"
+          echo "PLUGIN_INFO="$(<<<$PLUGIN_INFO jq '. + {user_id: ${{ steps.getuid.outputs.UID }} }')"" >> $GITHUB_ENV
+      - name: Get PR author email from GitHub username
+        id: getemail
+        uses: evvanErb/get-github-email-by-username-action@v1.25
+        with:
+          github-username: ${{github.event.pull_request.user.login}} # PR author's username
+          token: ${{ secrets.GITHUB_TOKEN }} # Including token enables most reliable way to get a user's email
+      - name: Add PR author email to PLUGIN_INFO
+        id: add_email_to_pluginfo
+        run: |
+          echo "The PR author email is ${{ steps.getemail.outputs.email }}"
+          echo "PLUGIN_INFO="$(<<<$PLUGIN_INFO jq '. + {author_email: ${{ steps.getemail.outputs.email }} }')"" >> $GITHUB_OUTPUT
+
+  runscore:
+    name: Score plugins
+    runs-on: ubuntu-latest
+    needs: changes_models_or_benchmarks
+    if: ${{ needs.changes_models_or_benchmarks.outputs.RUN_SCORE == 'True' }}
+    env:
+      PLUGIN_INFO: ${{ needs.get_submitter_info.outputs.PLUGIN_INFO }}
+      JENKINS_USR: ${{ secrets.JENKINS_USR }}
```

Comment:
what happened with the `E`? 😄

Reply:
Tight deadline, no time to type extraneous Es